language/stdlib/modules/libra_account.move

```diff
@@ -32,6 +33,11 @@ module LibraAccount {
         event_generator: EventHandleGenerator,
     }
 
+    // A resource that holds the libra coins stored in this account
+    resource struct Balance {
+        balance: LibraCoin::T,
```

Comment:
Nit: should this be called `value` or `coin`? The "balance.balance()" pattern that happens with this is a bit awkward....

Reply:
Yea, `balance.balance()` is odd...I'll update the field name to `coin`